monkey/tests/unit_tests/monkey_island/cc/repository/test_mongo_machine_repository.py

```diff
@@ -232,6 +232,25 @@ def test_get_machine_by_ip__retrieval_error(error_raising_machine_repository):
         error_raising_machine_repository.get_machines_by_ip("1.1.1.1")
 
 
+def test_get_machines(machine_repository):
+    retrieved_machines = machine_repository.get_machines()
+
+    for machine in MACHINES:
+        assert machine in retrieved_machines
+
+
+def test_get_machines__empty_repository(machine_repository):
+    machine_repository.reset()
```

Comment:
`test_get_new_id__new_id_for_empty_repo` also needs an empty repository. Add an `empty_machine_repository` fixture and use that for both tests.